tests/src/image.rs

```diff
@@ -11,7 +11,7 @@ async fn read_png(path: impl AsRef<Path>, width: u32, height: u32) -> Option<Vec
         Ok(f) => f,
         Err(e) => {
             log::warn!(
-                "image comparison invalid: file io error when comparing {}: {}",
+                "image comparison invalid: file io error when 3comparing {}: {}",
```

Comment:
typo?

Reply:
Fixed in https://github.com/gfx-rs/wgpu/pull/4765